admin/src/components/connect-banner.tsx

```diff
@@ -0,0 +1,21 @@
+import Image from '@atlaskit/image';
+import { css } from '@emotion/react';
+
+export function ConnectBanner() {
+	return (
+		<div
+			css={css`
+				display: flex;
+				align-items: center;
+			`}
+		>
+			<div css={css({ padding: 16 })}>
+				<Image src="/admin/jira-logo.svg" />
```

Comment:
Might be worth also testing this page and some of the assets with Jira's dark mode turned on if you haven't already. `<Image>` has a handy `srcDark` prop that can help if we end up also needing any dark mode assets

Reply:
I have not! Will add this to the test plan!

Reply:
gonna look into dark mode as a separate action item here. I'm noticing that while the app does support natively support dark mode, it inherits from the browser rather than from the atlassian context. So the rest of Jira may still be in light mode and the configure page is rendering in dark mode. Gonna look into whether there's a connect app JS api for determining this and go from there